server-spi/src/main/java/org/keycloak/models/IdentityProviderMapperModel.java

```diff
@@ -28,6 +34,10 @@
  */
 public class IdentityProviderMapperModel implements Serializable {
 
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
Don't create an ObjectMapper instance here. Instead use JsonSerialization.readValue as it creates a shared ObjectMapper instance.

Reply:
done